configurations/CAndy/dev/allowed-did.csv

```diff
@@ -0,0 +1,3 @@
+registered_did,details
+<CDT DID TBD>,Cyber Security and Digital Trust
+<CSB DID TBD>,Court Services Branch
```

Comment:
CDT and CSB DIDs to be defined as part of https://github.com/bcgov/DITP/issues/95